metisp/workflows/metis/metis_ifu_ravc_wkf.py

```diff
@@ -0,0 +1,12 @@
+from edps import SCIENCE, QC1_CALIB, QC0, CALCHECKER
+from edps import task, subworkflow, qc1calib, match_rules, FilterMode, calchecker
+from .metis_datasources import *
+from . import metis_keywords as metis_kwd
+from .metis_ifu_wkf import *
+
+ifu_ravc_post_task = (task('ifu_ravc_post')
```

Comment:
@eiseleb47 renamed all the tasks such that the names are consistent. That is, having each task name be the same as the recipe, optionally with a suffix. So it is immediately clear what recipe a task runs. Here this seems very much possible, e.g.:
```suggestion
metis_ifu_adi_racv_task = (task('metis_ifu_adi_racv')
```

Reply:
I need to rework the workflow a bit; the RAW science input for the workflow will need a different classification rule than the standard lm_img sequence to select only the correct type of HCI images, so importing the whole lm_img workflow won't work.